xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -113,6 +114,6 @@ class CGUIDialogVideoInfo :
 private:
   static std::string ChooseArtType(const CFileItem& item);
   static bool ManageVideoItemArtwork(const std::shared_ptr<CFileItem>& item,
-                                     const MediaType& type,
-                                     bool& finished);
```

Comment:
glad to see this going away :)

Reply:
I'm glad you spotted this improvement. ;-)